src/ctia/stores/es/query.clj

```diff
@@ -2,6 +2,34 @@
   (:require [clj-momo.lib.es.query :as q]
             [clojure.string :as str]))
 
+(def public-tlps
+  ["white" "green"])
+
+(defn find-restriction-query-part
+  [{:keys [login groups]}]
+  ;; TODO do we really want to discard case on that?
+  (let [login (str/lower-case login)
+        groups (map str/lower-case groups)]
+    {:bool
+     {:should
+      [{:terms {"tlp" public-tlps}}
+       ;; Document Owner
+       {:bool {:filter [{:term {"owner" login}}
+                        {:terms {"groups" groups}}]}}
```

Comment:
Does that means intersection is not empty?

Reply:
looks like it according to the docs:
https://www.elastic.co/guide/en/elasticsearch/reference/5.4/query-dsl-terms-query.html